manifests/04-rbac-rolebinding-cluster.yaml

```diff
@@ -54,3 +54,16 @@ subjects:
   - kind: ServiceAccount
     name: console
     namespace: openshift-console
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-chartrepos-view
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: 'system:authenticated'
```

Comment:
I believe you're need a PR like https://github.com/openshift/origin/pull/23975 to get this to pass CI.

Reply:
Pushed in https://github.com/openshift/origin/pull/25485